frontends/relay/relay_visitor.py

```diff
@@ -162,14 +178,49 @@ def visit_let(self, let):
                     dest=dest,
                     args=value.args,
                     attributes=value.attrs,
-                    data_type=ru.get_dahlia_data_type(let.var.type_annotation),
+                    data_type=ru.get_dahlia_data_type(type_annotation),
                 )
             )
         else:
             assert 0, f"{value} is not supported yet."
 
+    def visit_let(self, let):
+        """Visits a `let` statement in the following manner:
+        1. Visit the `value`.
+        2. Visit the `var`, or destination.
+        3. Return the `body`.
+        """
+        # Check if the dest is a tuple
+        if isinstance(let.var.type_annotation, tvm.ir.type.TupleType):
+            value = self.visit(let.value)
+            # Handles cases such as: `%x13 = (%x9, %x12)`. where %x9 and %x12 will
+            # evaluate to cells
+            assert isinstance(value, list) and len(value) == len(
+                let.var.type_annotation.fields), "Currently, if let destination is a tuple, can only handle 'tuple forwarding' situations"
+            unnested_values = []
+            # need to do this bc visit_var now returns a list
+            for dest in value:
+                assert isinstance(dest, list) and isinstance(
+                    dest[0], Cell), "Currently tuples in let value must evaluate to cells"
+                unnested_values.append(dest[0])
+            # doesn't do anything just increments id by 1 so that we can
```

Comment:
So that we can ___ ?

Reply:
Ah, I wanted to say that "so we can compare the Calyx files with the generated TVM relay more easily". If I didn't do this, the names of the variables in TVM relay and Calyx would get off by one .  This was mainly to help me when I was doing the "tuple forwarding" stuff to make sure I was "forwarding" the tuples correctly.  I should probably update that comment.